packages/appstore/src/components/wallet-transfer/wallet-transfer.tsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
-import { AmountInput, TransferAccountSelector } from '@deriv/components';
+import { Field, FieldProps, Formik, Form } from 'formik';
```

Comment:
@heorhi-deriv just a suggestion, please order the imports by following this [format](https://github.com/binary-com/deriv-app/tree/master/docs/JavaScript#import-rules) 🙂.

Reply:
hey @aum-deriv what should be the order regarding this link?
seems like it already sorted correctly